src/commands/index.ts

```diff
@@ -539,77 +539,81 @@ export class VSCodeCommands implements Disposable {
           this.dbtTerminal.logLine(`Error=${e}`);
         }
       }),
-      commands.registerCommand("dbtPowerUser.createSqlFile", async () => {
-        try {
-          const project =
-            await this.queryManifestService.getOrPickProjectFromWorkspace();
-          if (!project) {
-            window.showErrorMessage("No dbt project selected.");
-            return;
-          }
+      commands.registerCommand(
+        "dbtPowerUser.createSqlFile",
+        async ({ code, fileName }: { code?: string; fileName?: string }) => {
```

Comment:
added these 2 arguments to display selected code from bookmark or history details and with filename